test-fixtures/removeBlob/01/expectedResult.json

```diff
@@ -0,0 +1,4 @@
+{
+  "blobmetadatas": [],
+  "blobmetadatas.files": []
```

Comment:
In api client removing blob does not remove blob content. Is there a error somewhere if this is desired by the test?

Reply:
Otherwise, consider including the file within the test to verify that the file is not deleted even if blob is.

Reply:
In the api is prevention to delete blob unless content is deleted first. This makes sure that no content get orphaned in the db. This change is to make blob removal simpler and keeps DB clean.

https://github.com/NatLibFi/melinda-record-import-api/blob/e02de5167285d75bc136ff2e9064920d3ba70da5/src/interfaces/blobs.js#L186